WordPress/gutenberg
# Add a button to allow resetting the ComboboxControl value

When using the ComboboxControl in selects where "null" values are allowed like #25267 we need to be able to reset its value.
This PR does so by adding a reset button.

**Testing instructions**

 - npm run storybook:dev
 - Check the ComboboxControl story and see that you can choose countries and reset the value as well.
